tests/models/rembert/test_tokenization_rembert.py

```diff
@@ -0,0 +1,162 @@
+# coding=utf-8
+# Copyright 2022 The HuggingFace Team. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+""" Testing suite for the RemBert tokenizer. """
+
+
+import unittest
+
+from transformers import RemBertTokenizer, RemBertTokenizerFast
+from transformers.testing_utils import get_tests_dir, require_sentencepiece, require_tokenizers
+
+from ...test_tokenization_common import TokenizerTesterMixin
+
+
+SENTENCEPIECE_UNDERLINE = "▁"
+SPIECE_UNDERLINE = SENTENCEPIECE_UNDERLINE  # Kept for backward compatibility
+
+SAMPLE_VOCAB = get_tests_dir("fixtures/test_sentencepiece.model")
+
+
+@require_sentencepiece
+@require_tokenizers
+class RemBertTokenizationTest(TokenizerTesterMixin, unittest.TestCase):
+    tokenizer_class = RemBertTokenizer
+    rust_tokenizer_class = RemBertTokenizerFast
+    space_between_special_tokens = True
+    test_rust_tokenizer = True
+    test_sentencepiece_ignore_case = True
+    pre_trained_model_path = "google/rembert"
+
+    def setUp(self):
+        super().setUp()
+
+        tokenizer = RemBertTokenizer(SAMPLE_VOCAB)
+        tokenizer.save_pretrained(self.tmpdirname)
+
+    def get_input_output_texts(self, tokenizer):
+        input_text = "this is a test"
+        output_text = "this is a test"
+        return input_text, output_text
+
+    def test_convert_token_and_id(self):
```

Comment:
not sure this one is needed!

Reply:
If you think its trivial we can remove it. Let me know.